src/Sylius/Bundle/ApiBundle/Resources/config/services/data_providers.xml

```diff
@@ -75,6 +75,11 @@
             <tag name="api_platform.collection_data_provider" priority="10" />
         </service>
 
+        <service id="Sylius\Bundle\ApiBundle\DataProvider\ChannelsCollectionDataProvider">
+            <argument type="service" id="Sylius\Component\Channel\Context\ChannelContextInterface" />
```

Comment:
```suggestion
            <argument type="service" id="sylius.context.channel" />
```
if I see correctly we are using this name of service  (but i know that it is an alias for `"Sylius\Component\Channel\Context\ChannelContextInterface"`